docs/release-notes/1.0.0/release-1.0.0.md

```diff
@@ -0,0 +1,80 @@
+# ML.NET 1.0.0 Release Notes
+
+[ML.NET](https://aka.ms/mlnet) is now 1. :cake:. After 12 preview releases over the last year and implementing many enhancements and API improvements, this release brings you the version `1.0.0` of [ML.NET](https://aka.ms/mlnet). Thank you wholeheartedly for being an awesome comunity. Please continue to use and provide feedback on what should be improved. We will ensure that going forward no breaking API changes are introduced.
```

Comment:
>Thank you wholeheartedly [](start = 230, length = 24)

Thank you, wholeheartedly

Reply:
This would be an unnecessary punctuation.